server/routes/referrals/referralStartPresenter.ts

```diff
@@ -1,7 +1,12 @@
 import { DraftReferral } from '../../services/interventionsService'
+import { FormValidationError } from '../../utils/formValidationError'
+import ReferralDataPresenterUtils from './referralDataPresenterUtils'
 
-export default class DraftReferralsListPresenter {
-  constructor(private readonly draftReferrals: DraftReferral[]) {}
+export default class ReferralStartPresenter {
+  constructor(
+    private readonly draftReferrals: DraftReferral[],
+    private readonly errors: FormValidationError | null = null
```

Comment:
```suggestion
    private readonly error: FormValidationError | null = null
```

(there's one other place in the codebase where we've called it `errors` by mistake, I'll make a PR to fix it)

Reply:
Fixed